### ISSUE TITLE
How to set the precision of the back propagation gradient during quantified training in the brevitas library？

### ISSUE THREAD

**Issue by lpy980429** [Xilinx/brevitas#382]
I want to use the brevitas library for quantization training, and I want to set the quantization precision of the gradient, what should I do?
I only construct the forward network which can set quantization precision of every layers. When back propagation, I only use loss.backward() to do it. I can't see back propagation mechanism, please help me. Thanks.
